packages-content-model/roosterjs-content-model-editor/lib/modelApi/common/mergeModel.ts

```diff
@@ -287,7 +287,8 @@ function splitParagraph(markerPosition: InsertPoint, newParaFormat: ContentModel
 
 function insertBlock(markerPosition: InsertPoint, block: ContentModelBlock) {
     const { path } = markerPosition;
-    const newPara = splitParagraph(markerPosition, block.format);
+    const newParaFormat = block.blockType === 'Table' ? {} : block.format;
```

Comment:
I feel we should get this format only when block is paragraph. For all other block types, such as ListItem, Table, Divider, Entity, we should not inherit its format. What do you think?

Reply:
I agree with you.  I was not too sure to handle those block types too in this PR. But now that you mention it I will add that change too.